#The header of the source image
Header header

DetectedObject[] objects